ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/PackageConfigurationFactory.kt

```diff
@@ -25,7 +26,9 @@ internal object PackageConfigurationFactory {
             }
             rcPackage
         }
-        require(filteredRCPackages.isNotEmpty()) { "No packages found for ids $filter" }
+        if (filteredRCPackages.isEmpty()) {
+            return Result.failure(PackageConfigurationError("No packages found for ids $filter"))
```

Comment:
I do wonder if we should fallback to the default configuration in this case... But for now, I think it's ok to just display an error.

Reply:
That's a good point, but which package do we pick? The first one available?

Reply:
Hmm yeah... I guess the current if it exists and if not, fallback to the first... but I think this is also ok